test/core/xds/xds_transport_fake.cc

```diff
@@ -120,15 +126,50 @@ void FakeXdsTransportFactory::FakeStreamingCall::CompleteSendMessageFromClient(
   CompleteSendMessageFromClientLocked(ok);
 }
 
+void FakeXdsTransportFactory::FakeStreamingCall::StartRecvMessage() {
+  absl::optional<std::string> pending;
```

Comment:
This variable appears to be unused.  I think it can be removed.

Reply:
Done